irteus/compile_irtg.l

```diff
@@ -31,6 +31,8 @@
 
 (comp:compile-file-if-src-newer "irtgeo.l" user::*objdir*)
 (comp:compile-file-if-src-newer "pqp.l" user::*objdir*)
+(if (string= (unix:getenv "BULLET_GE_2_83") "true")
```

Comment:
I think compile eusbullet.l for all cases is ok. just to 
```
void BulletcalcCollisionDistance() {
#if HAVE_BULLET
#else
    fpritnf(stderr, "jskeus is compile without bullet, so you can not use function %s, Please install bullet > 2.83\n", __PRETTY_FUNCTION__); or something
#endif
}
```

Reply:
Fixed in https://github.com/euslisp/jskeus/pull/538/commits/2a89385f61049c9f4e08400150979aa6860fd6c4.

Now, compile eusbullet.l and BulletCollision.cpp even if bullet is not installed, and check bullet installed as follows: https://github.com/euslisp/jskeus/blob/2a89385f61049c9f4e08400150979aa6860fd6c4/irteus/BULLET/BulletCollision.cpp#L56-L60

Not smart point is that I should comment out some parts with macro's `if` to avoid compile error of undefined function:
https://github.com/euslisp/jskeus/blob/2a89385f61049c9f4e08400150979aa6860fd6c4/irteus/BULLET/BulletCollision.cpp#L244-L257